pkg/controller/BUILD.bazel

```diff
@@ -61,6 +61,7 @@ go_library(
         "//vendor/k8s.io/client-go/tools/cache:go_default_library",
         "//vendor/k8s.io/client-go/tools/record:go_default_library",
         "//vendor/k8s.io/klog/v2:go_default_library",
+        "//vendor/k8s.io/utils/pointer:go_default_library",
```

Comment:
I think this needs another `make deps-update` since it is marked "indirect" in go.mod